src/View/Event/ViewEvent.php

```diff
@@ -8,15 +8,15 @@
 abstract class ViewEvent
 {
     /**
-     * @var string the view file being rendered.
+     * @var string|bool the view file being rendered.
      */
     private $file;
     /**
      * @var array the parameter array passed to the [[View::render()]] method.
      */
     private $parameters;
 
-    public function __construct(string $file, array $parameters = [])
+    public function __construct($file, array $parameters = [])
```

Comment:
This change doesn't seem to be related.

Reply:
Phpstan suggests that it can be string or bool, we should stop working with variables of multiple types.

Reply:
Bool? Why?

Reply:
I'll check it.